evap/staff/importers/user.py

```diff
@@ -385,5 +385,8 @@ def update_existing_and_create_new_user_profiles(
     existing_user_profiles: Iterable[UserProfile],
     new_user_profiles: Iterable[UserProfile],
 ):
-    UserProfile.objects.bulk_update(existing_user_profiles, UserData.bulk_update_fields())
```

Comment:
I suppose `bulk_update_fields` is not needed anymore then? The word "bulk" also occurs in the comment of `UserData.get_user_profile_object`

Reply:
Yes. This also applies to the whole `LoggedModel.update_log_after_...` stuff, which I added in #1634 to allow for bulk operations here.

For now, I thought it would be best to leave this in, wait for what we think is a good solution to the general save-sideeffects-vs-bulk-operations problem in #1839, and then maybe (or maybe not) remove them.